samples/sample_capture_assistant.py

```diff
@@ -10,7 +10,7 @@ def _main():
 
     suggest_settings_parameters = SuggestSettingsParameters(
         max_capture_time=datetime.timedelta(milliseconds=1200),
-        ambient_light_frequency=SuggestSettingsParameters.AmbientLightFrequency.none,
+        ambient_light_frequency="none",
```

Comment:
Why was this changed?

Reply:
See the relevant commit message:
```
I believe this is better, as it makes using the CA seem easier.  Strings
are the common way of inputting such arguments in Python anyway.
```

I don't feel strongly about this though, I have just found myself being annoyed at typing out a lot of stuff lately when wanting to use the CA in python, and just realized that they are strings now. 
Feel free to disagree, or say that it doesn't belong in this PR :)

Reply:
It might be easier to discover the other values ambient light frequenct could be if there with the previous solution. If a user just wants to play around with this sample and wants to change the frequency to something else, the user`s IDE could potentially help them out.

Reply:
Good point. I can revert this.

Reply:
Done.